R/utilities.R

```diff
@@ -2553,11 +2570,11 @@ massage_matrix_for_clustering = function(incoming_data,
   # did user provide any features they would like to drop from matrix?
   if(!missing(drop_these_features)){
     message("You provided features to be dropped from matrix, removing them ...")
-    output_data = 
```

Comment:
Is there a way to automatically remove spaces, I see you did quite a few edits of that kind in this script?

Reply:
There is! Very handy! I just started doing this. Here is the Slack thread where we discussed it: https://morinlabsfu.slack.com/archives/C0224H120CU/p1648871548155479?thread_ts=1648836087.433839&cid=C0224H120CU

Reply:
Yes, I remember that discussion. Neat!